### PR TITLE
Add models

### PR DESCRIPTION
**Pull request by cristianbgp**

### CODE REVIEW

File: app/models/booking.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Booking < ApplicationRecord
+  belongs_to :user
```
Comment (byteconde) [inline app/models/booking.rb]: Great

File: app/models/promotion.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class Promotion < ApplicationRecord
+  belongs_to :promotionable, polymorphic: true
```
Comment (byteconde) [inline app/models/promotion.rb]: Poly added in promotion and associated with hotel and room

File: db/migrate/20190429210537_create_rooms.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class CreateRooms < ActiveRecord::Migration[5.2]
+  def change
+    create_table :rooms do |t|
+      t.string :name
+      t.integer :amount_of_beds
+      t.integer :price
+      t.references :hotel, foreign_key: true
```
Comment (byteconde) [inline db/migrate/20190429210537_create_rooms.rb]: Room associated with hotel

File: db/migrate/20190429210550_create_bookings.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+class CreateBookings < ActiveRecord::Migration[5.2]
+  def change
+    create_table :bookings do |t|
+      t.date :start_date
+      t.date :end_date
+      t.integer :paid_price
+      t.references :user, foreign_key: true
```
Comment (byteconde) [inline db/migrate/20190429210550_create_bookings.rb]: Booking associated with user and room

File: db/migrate/20190429210836_add_role_to_users.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :role, :string
```
Comment (byteconde) [inline db/migrate/20190429210836_add_role_to_users.rb]: Role added

File: app/models/booking.rb (hunk 1 above)
Comment (liannivin) [reply app/models/booking.rb]: You had a little bug with your class's name

**Pull request merged**